attest/enclave-api/src/error.rs

```diff
@@ -61,15 +62,15 @@ pub enum Error {
      * Invalid state for call
      *
      * This indicates a bug in the calling code, typically attempting to
-     * re-submit an already-verified quote or IAS report.
+     * re-submit an already-verified quote or attestation evidence.
      */
     InvalidState,
 
-    /// No IAS report has been verified yet
-    NoReportAvailable,
+    /// No attestation evidence has been verified yet
+    NoAttestationEvidenceAvailable,
 
-    /// Too many IAS reports are already in-flight
-    TooManyPendingReports,
+    /// Too many attestation evidence instances are already in-flight
+    TooManyPendingAttestationEvidences,
```

Comment:
Up for suggestions on this name "evidences" feels awkward

Reply:
`TooMuchPendingAttestationEvidence` sounds nicer but at the expense of being less correct.

Reply:
I think it's probably fine. But you could also do the more verbose `TooManyPendingAttestationEvidenceInstances` or just `TooManyPendingEvidenceInstances`

Reply:
Updated to `TooManyPendingEvidenceInstances`, also standardized the name in `fog/ingest/enclave/api/src/error.rs`